demonstrations/learning2learn.py

```diff
@@ -0,0 +1,1205 @@
+r"""
+Learning to Learn with Quantum NN via Classical NN
+==================================================
+*Author: Stefano Mangini (mangini.stfn@gmail.com)*
+
+.. meta::
+    :property="og:description": Use a classical recurrent neural network to initilize the parameters of a variational quatum algorithm.
+    :property="og:image": ../demonstrations/learning2learn/thumbnail.png
+
+.. related::
+
+   tutorial_qaoa_intro QAOA
+   tutorial_qaoa_maxcut QAOA for MaxCut problem
+
+
+In this tutorial we recreate the architecture proposed by Verdon et
+al. in the paper *Learning to learn with quantum neural networks via
```

Comment:
```suggestion
al. in *Learning to learn with quantum neural networks via
```
to avoid repeated use of "paper"